client/go/dogma/cmd/cmd.go

```diff
@@ -0,0 +1,271 @@
+// Copyright 2018 LINE Corporation
+//
+// LINE Corporation licenses this file to you under the Apache License,
+// version 2.0 (the "License"); you may not use this file except in compliance
+// with the License. You may obtain a copy of the License at:
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations
+// under the License.
+
+// Package cmd implements commands that execute HTTP API to download, add, create, modify, or delete the
+// specified resources on the Central Dogma server.
+// Commands are created according to the input from the CLI and executed.
+package cmd
+
+import (
+	"bufio"
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+	"os"
+	"strings"
+	"syscall"
+
+	"github.com/line/centraldogma/client/go/dogma"
+	"github.com/urfave/cli"
+	"golang.org/x/crypto/ssh/terminal"
+)
+
+// Command is the common interface implemented by all commands.
+type Command interface {
+	// execute executes the command
+	execute(c *cli.Context) error
+}
+
+func getRemoteURL(remoteURL string) (string, error) {
+	if len(remoteURL) == 0 {
+		fmt.Print("Enter specific address to connect: ")
```

Comment:
- `Enter server address:`?
- How about adding an example for all prompts? e.g. `Enter server address: (e.g. example.com:36462)\n`

Reply:
I think adding an example to this prompt is really good idea.
There are three other prompts which are getting username, password and summary.
I don't think they don't need a example. Can I just add it only here?

Reply:
SGTM